xray/audit/java/javautils.go

```diff
@@ -25,21 +25,15 @@ type DependencyTreeParams struct {
 	JavaProps        map[string]any
 }
 
-func createBuildConfiguration(buildName string) (*artifactoryUtils.BuildConfiguration, func(err error)) {
+func createBuildConfiguration(buildName string) (*artifactoryUtils.BuildConfiguration, func() error) {
 	buildConfiguration := artifactoryUtils.NewBuildConfiguration(buildName, strconv.FormatInt(time.Now().Unix(), 10), "", "")
-	return buildConfiguration, func(err error) {
-		buildName, err := buildConfiguration.GetBuildName()
-		if err != nil {
-			return
-		}
+	return buildConfiguration, func() (err error) {
```

Comment:
The function does not use defers. I think there is no need to use`(err error)` here